support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -291,27 +272,13 @@ object DigitalSubscriptionGiftRedemption {
   ): Future[HandlerResult[SendThankYouEmailState]] =
     zuoraService.getSubscriptionFromRedemptionCode(redemptionData.redemptionCode).flatMap(
       redemptionQueryResponse =>
-        getSubscriptionState(redemptionQueryResponse, state.requestId.toString) match {
+       GiftRedemptionState.getSubscriptionState(redemptionQueryResponse, state.requestId.toString) match {
```

Comment:
```suggestion
        GiftRedemptionState.getSubscriptionState(redemptionQueryResponse, state.requestId.toString) match {
```

Reply:
in case it's not clear I was adding the space back in above!

Reply:
not from this PR, but getSubscriptionState could take a UUID instead of a String.  This means that we get better localised type checking, so if we later change state.requestId to another type, we will get a compile error, rather than it running toString on whatever the new type is.  I'm not sure if our tests would cover that adequately.